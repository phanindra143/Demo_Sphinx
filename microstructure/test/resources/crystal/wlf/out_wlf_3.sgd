*if,ifprop,eq,2,then
  !c_ij(1,1)=     0.0000
  !c_ij(1,2)=     0.0000
  !c_ij(2,2)=     0.0000
  !c_ij(1,3)=     0.0000
  !c_ij(2,3)=     0.0000
  !c_ij(3,3)=     0.0000
  !c_ij(1,4)=     0.0000
  !c_ij(2,4)=     0.0000
  !c_ij(3,4)=     0.0000
  !c_ij(4,4)=     0.0000
  !c_ij(1,5)=     0.0000
  !c_ij(2,5)=     0.0000
  !c_ij(3,5)=     0.0000
  !c_ij(4,5)=     0.0000
  !c_ij(5,5)=     0.0000
  !c_ij(1,6)=     0.0000
  !c_ij(2,6)=     0.0000
  !c_ij(3,6)=     0.0000
  !c_ij(4,6)=     0.0000
  !c_ij(5,6)=     0.0000
  !c_ij(6,6)=     0.0000
*endif
*if,ifprop,eq,1,then
  !k_ij(1,1)=   132.3568
  !k_ij(1,2)=     0.0183
  !k_ij(2,2)=   132.2144
  !k_ij(1,3)=    -0.0180
  !k_ij(2,3)=     0.2837
  !k_ij(3,3)=   131.7121
MP,Kxx,1,  132.35683
MP,Kyy,1,  132.21444
MP,Kzz,1,  131.71212
MP,Kxx,2,  132.21444
MP,Kyy,2,  132.35683
MP,Kzz,2,  131.71212
MP,Kxx,3,  132.21444
MP,Kyy,3,  132.35683
MP,Kzz,3,  131.71212
MP,Kxx,4,  132.35683
MP,Kyy,4,  132.21444
MP,Kzz,4,  131.71212
MP,Kxx,5,  132.35683
MP,Kyy,5,  132.21444
MP,Kzz,5,  131.71212
MP,Kxx,6,  132.21444
MP,Kyy,6,  132.35683
MP,Kzz,6,  131.71212
MP,Kxx,7,  132.21444
MP,Kyy,7,  132.35683
MP,Kzz,7,  131.71212
MP,Kxx,8,  132.35683
MP,Kyy,8,  132.21444
MP,Kzz,8,  131.71212
MP,Kxx,9,  131.71212
MP,Kyy,9,  132.35683
MP,Kzz,9,  132.21444
MP,Kxx,10,  132.35683
MP,Kyy,10,  131.71212
MP,Kzz,10,  132.21444
MP,Kxx,11,  132.35683
MP,Kyy,11,  131.71212
MP,Kzz,11,  132.21444
MP,Kxx,12,  131.71212
MP,Kyy,12,  132.35683
MP,Kzz,12,  132.21444
MP,Kxx,13,  131.71212
MP,Kyy,13,  132.35683
MP,Kzz,13,  132.21444
MP,Kxx,14,  132.35683
MP,Kyy,14,  131.71212
MP,Kzz,14,  132.21444
MP,Kxx,15,  132.35683
MP,Kyy,15,  131.71212
MP,Kzz,15,  132.21444
MP,Kxx,16,  131.71212
MP,Kyy,16,  132.35683
MP,Kzz,16,  132.21444
MP,Kxx,17,  132.21444
MP,Kyy,17,  131.71212
MP,Kzz,17,  132.35683
MP,Kxx,18,  131.71212
MP,Kyy,18,  132.21444
MP,Kzz,18,  132.35683
MP,Kxx,19,  131.71212
MP,Kyy,19,  132.21444
MP,Kzz,19,  132.35683
MP,Kxx,20,  132.21444
MP,Kyy,20,  131.71212
MP,Kzz,20,  132.35683
MP,Kxx,21,  132.21444
MP,Kyy,21,  131.71212
MP,Kzz,21,  132.35683
MP,Kxx,22,  131.71212
MP,Kyy,22,  132.21444
MP,Kzz,22,  132.35683
MP,Kxx,23,  131.71212
MP,Kyy,23,  132.21444
MP,Kzz,23,  132.35683
MP,Kxx,24,  132.21444
MP,Kyy,24,  131.71212
MP,Kzz,24,  132.35683
*endif
