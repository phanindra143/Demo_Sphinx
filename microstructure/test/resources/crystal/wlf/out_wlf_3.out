            ifx                 q_x                 q_y                 q_z     
  /% source: R:/166_320/Ablage_Simulationen/Modellbildung_Simulation_CMC-SiC_[Vorlauf+WiMi+MaVo]/04_04_2017_Modellierung_Matrix_mikro_ditt_[WiMi]/2_Waermeleitfähigket_SiC-Si\Rein_ohne_Poren\60-40
             3.00000      -2068075.19283          1442.64301           566.86742
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             3.00000         15625.00000           -13.42189            -1.38814
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             2.00000          4531.05390      -2065824.42089          7308.17511
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             2.00000           -36.06990         15625.00000           -89.38602
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             1.00000           666.97282          7816.31383      -2057975.87910
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             1.00000            -3.65148           -92.39985         15625.00000
  /%
